libs/design/src/atoms/form/radio/radio.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { DaffRadioComponent } from './radio.component';
+import { Component, DebugElement } from '@angular/core';
+import { By } from '@angular/platform-browser';
+@Component({
+  template: `
+    <daff-radio [checked]=true value="testValue"></daff-radio>
+  `
+})
+class RadioWrapperComponent {}
+
+describe('DaffRadioComponent', () => {
+  let wrapper: RadioWrapperComponent;
+  let component: DaffRadioComponent;
+  let fixture: ComponentFixture<RadioWrapperComponent>;
+  let de: DebugElement;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        DaffRadioComponent,
+        RadioWrapperComponent
+      ]
+    })
+      .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(RadioWrapperComponent);
+    wrapper = fixture.componentInstance;
+    de = fixture.debugElement.query(By.css('daff-radio'));
+    component = fixture.debugElement.query(By.css('daff-radio')).componentInstance;
+    fixture.detectChanges();
+  });
+
```

Comment:
Needs tests describing with &without radioset

Reply:
Testing coverage expanded in 7eff94fc15e2ee0f2adf31c5de274475d7e0c73d. Let me know if it could cover more.